_assets/javascripts/vendor.js

```diff
@@ -2,3 +2,4 @@
 //= require throttle-debounce-fn/dist/throttle-debounce-fn.js
 //= require fluidbox
 //= require retinajs/dist/retina.js
+//= require svgxuse/svgxuse.js
```

Comment:
Prefer to leave this as is so that jekyll-assets and Yarn will always fetch and build vendor.js with the version specified in the package.json file. Otherwise, moving this into a browser dependant script src call will require the svgxuse.js be manually placed into the 'javascripts' folder so that jekyll-assets will still be able to source the file during `jekyll build`.